cf-net/cf-net.c

```diff
@@ -912,8 +909,14 @@ static int CFNetOpenDir(ARG_UNUSED CFNetOptions *opts, const char *hostname, cha
 
     const char *remote_path = args[1];
 
-    Item *items = RemoteDirList(remote_path, false, conn);
-    PrintDirs(items);
+    Seq *seq = CfNetOpenDir(conn, remote_path);
+    if (seq == NULL)
+    {
+        return -1;
```

Comment:
Is this value propagated as the exit code of the whole process? If yes, then it should not be `-1`, but rather `1` or something similar.

Reply:
`-1` is used throughout the rest of `cf-net` as well, though. I think a better change would be to check the result of functions and using correct exit code based on that. Seems arbitrary for a function to return `EXIT_FAILURE` for instance.

Reply:
Cannot agree more. Just please do the change in a follow-up commit or create a task so that it gets fixed later.